backend/src/Controller/CaptainFinancialSystem/CaptainFinancialSystemAccordingToCountOfOrdersController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Controller\CaptainFinancialSystem;
+
+use App\Controller\BaseController;
+use App\Service\CaptainFinancialSystem\CaptainFinancialSystemAccordingToCountOfOrdersService;
+use Nelmio\ApiDocBundle\Annotation\Security;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use OpenApi\Annotations as OA;
+
+/**
+ * Create and fetch Captain's Financial System According To Count Of Orders.
+ * @Route("v1/captainfinancialsystemaccordingntocountoforders/")
+ */
+class CaptainFinancialSystemAccordingToCountOfOrdersController extends BaseController
+{
+    private CaptainFinancialSystemAccordingToCountOfOrdersService $CaptainFinancialSystemAccordingToCountOfOrdersService;
```

Comment:
update the var name (can not start with capital letter), and change it also in all places in which it is being used.